openpype/lib/applications.py

```diff
@@ -1302,10 +1302,18 @@ def _prepare_last_workfile(data, workdir):
     )
     data["start_last_workfile"] = start_last_workfile
 
+    workfile_startup = should_workfile_tool_start(
```

Comment:
I think that if workfiles tool should be shown on startup then `start_last_workfile` should be `False`, or?

Reply:
I agree and I have noticed the collision it creates. But I believe anyone who will be setting this up will do this himself. At least that's how I understand the explicit approach.

Reply:
Or maybe not and let host handle that...